### ISSUE TITLE
Memory leak of facemesh javascript solution with iris detection

### ISSUE THREAD

**Issue by shirlyyang-tcc** [google/mediapipe#3678]
**System information** (Please provide as much relevant information as possible)
- no
- Chrome 105.0.5195.102
- [0.4.1629159166](https://www.npmjs.com/package/@mediapipe/face_mesh/v/0.4.1629159166)
- Bazel version:no
- Solution (e.g. FaceMesh, Pose, Holistic):FaceMesh
- Programming Language and version ( e.g. C++, Python, Java):JavaScript

I am developing a AR application on Web and recently we discovered that facemesh has the problem of memory leaking in version 0.4.1629159166 and 0.4.1633559619. And I tested version 0.4.1628005669 it does not have the same problem. But my application do need iris data while version  0.4.1628005669 does not support. 

I checked the code that initialize() is called only once and image is sent after last onResult() was triggered. 

I alse tested in offical demo (https://codepen.io/mediapipe/details/KKgVaPJ) and it causes memory leaks too. The image is the heapshot of facemesh in offical demo in which snapshot2 is taken 2 minutes after  snapshot1. 
<img width="243" alt="企业微信截图_c2128243-721e-437b-961d-c5ae4bd72d45" src="https://user-images.githubusercontent.com/69131160/189025077-4202beeb-eebe-4393-b62c-d19bb198123c.png">
As can be seen the main increase comes from HEAP32 arraybuffer.
<img width="978" alt="image" src="https://user-images.githubusercontent.com/69131160/189025419-b6d7b496-330d-4df3-913d-2b2ccf45d2f0.png">
<img width="990" alt="企业微信截图_a085eed7-0aa3-40d8-8058-75ad33ce42d8" src="https://user-images.githubusercontent.com/69131160/189025546-89e70b2f-6897-4d18-b4bf-b6909212895d.png">

Is there any way to solve this problem in the iris versions or is it possible? Really urgent and looking forward to responses.

**Comment by kuaashish** [comment]
Hi @shirlyyang-tcc, Have you looked at this similar closed issue #1937 and [#1937(comment)](https://github.com/google/mediapipe/issues/1937#issuecomment-833135989). Please try the suggestion and revert back if further queries. Thank you!

**Comment by shirlyyang-tcc** [comment]
> Hi @shirlyyang-tcc, Have you looked at this similar closed issue #1937 and [#1937(comment)](https://github.com/google/mediapipe/issues/1937#issuecomment-833135989). Please try the suggestion and revert back if further queries. Thank you!

Yes I did read this issue and tryed the suggestions in the issue. I did check the code and reassured the face mesh was not initialize multiple times. An I tested the versions decribed in this response:https://github.com/google/mediapipe/issues/1937#issuecomment-1039587436. 
As it describes versions 0.4.1629159166 and now 0.4.1633559619 both have memory leaks, version 0.4.1628005669 does not have the problem but it does not support iris detection. Our application needs iris detection so we can't use version 0.4.1628005669. So the problem still exists.
